spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -48,7 +48,7 @@
 	 * duplicated if the proxy and the backend are secured with Spring. By default they
 	 * are added to the ignored headers if Spring Security is present.
 	 */
-	private static final List<String> SECURITY_HEADERS = Arrays.asList("Pragma",
+	public static final List<String> SECURITY_HEADERS = Arrays.asList("Pragma",
```

Comment:
Adding Spring Security changes the headers injected by Zuul. `ZuulPropertiesTests.defaultIngoredHeaders` started to fail for this reason. I updated the tests to expect the headers declared with `ZuulProperties.SECURITY_HEADERS`.